### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by ConnorKirk** [mint-lang/mint#398]
Hi there!

I was trying to install mint-ui to try it out.

When I add the dependency and run `mint install` I receive the following error.
```sh
I could not checkout the version or tag: 1.1.0 of the repository: 
https://github.com/mint-lang/mint-ui 

The error I got from the git command is this: 

error: Your local changes to the following files would be overwritten by 
checkout:
        README.md
Please commit your changes or stash them before you switch branches.
Aborting 
``` 
I'm still new to mint, so I'm not 100% confident that this is not user error. I am able to successfully install `mint-color` however.

**Comment by gdotdesign** [comment]
I've added the `-f` (force flag) to the git checkout command when installing in https://github.com/mint-lang/mint/commit/f78268f77e03f152881d31cd778c6fa774d7a9ad so this in theory should fix the issue so if you would try it and confirm it would be good! You can find the binaries here https://github.com/mint-lang/mint/actions/runs/758310781 under `Artifacts`.

**Comment by ConnorKirk** [comment]
I can confirm this worked. 🎉 

Thanks for all of your help @gdotdesign  I really appreciate it.

**Comment by gdotdesign** [comment]
Great! :smile: I'll close the issue then and it will be released in `0.13.0`
